site/content/en/blog/releases/0.7.0.md

```diff
@@ -44,7 +44,7 @@ Images available with this release:
 - [gcr.io/agones-images/agones-controller:0.7.0](https://gcr.io/agones-images/agones-controller:0.7.0)
 - [gcr.io/agones-images/agones-sdk:0.7.0](https://gcr.io/agones-images/agones-sdk:0.7.0)
 - [gcr.io/agones-images/cpp-simple-server:0.4](https://gcr.io/agones-images/cpp-simple-server:0.4)
-- [gcr.io/agones-images/udp-server:0.5](https://gcr.io/agones-images/udp-server:0.5)
+- [gcr.io/agones-images/udp-server:0.6](https://gcr.io/agones-images/udp-server:0.6)
```

Comment:
ooh, we should get rid of this - this is an old blog post

Reply:
fixed that.